packages/react/src/internal.tsx

```diff
@@ -1,6 +1,7 @@
 export * from './hooks/useAuth';
 export * from './hooks/useDataStore';
 export * from './hooks/useStorageURL';
+export * from './hooks/actions/useAuthSignOutAction';
```

Comment:
Should we start using explicit export?

Reply:
Yeah, that's probably safer. Will update